anycloud/node/src/datastore.ts

```diff
@@ -1,32 +1,67 @@
-import { dsdel, dsgetv, dshas, dssetv } from 'alan-js-runtime';
-
-const ns = 'kv';
-
-export const DS = new Proxy({}, {
-  get: function (_, dsKey) {
-    const dsValue = dsgetv(ns, dsKey);
-    return dsValue.length === 2 ? dsValue[1] : 'Value not found';
-  },
-  set: function (_, dsKey, dsValue) {
-    if (dshas(ns, dsKey)) { return false; }
-    dssetv(ns, dsKey, dsValue)
-    return true;
-  },
-  deleteProperty: function (_, dsKey) {
-    if (!(dshas(ns, dsKey))) { return false; }
-    return dsdel(ns, dsKey);
-  },
-  has: function (_, dsKey) {
-    return dshas(ns, dsKey);
-  },
-});
-
-
-console.log(DS['foo'] = 'bar');
-console.log(DS['foo']);
-console.log('foo' in DS);
-console.log(delete DS['foo']);
-console.log('foo' in DS);
-console.log(DS['foo'] = {foo1: "bar1"});
-console.log(DS['foo']);
-console.log(delete DS['foo']);
+require('cross-fetch/polyfill')
+
+class DataStore {
+  private localDS: any;
+  private ns: string;
+  private clusterSecret?: string;
+  private isLocal: boolean;
+  private headers?: any;
+  private ctrlPortUrl: string;
+
+  constructor() {
+    this.localDS = {};
+    this.ns = 'kv';
```

Comment:
Nit: Maybe this field should be an optional constructor argument? Not necessary for now, but maybe in the future when we want to power other things with Datastore, too.

Reply:
I've remove it since we handle this right now on `anycloud.ln` event handler. I think it would make sense to add it once we need it?

Reply:
That's probably even better; put as much of that into the control port logic on the anycloud.ln file so users can't accidentally break it.